docs/mdbook/src/android/map-options-guide.md

```diff
@@ -185,9 +183,7 @@ class MapOptionsRuntimeActivity : AppCompatActivity(), OnMapReadyCallback {
 
     override fun onMapReady(maplibreMap: MapLibreMap) {
         this.maplibreMap = maplibreMap
-        this.maplibreMap.setStyle(
-            Style.Builder().fromUri(TestStyles.getPredefinedStyleWithFallback("Streets"))
-        )
+        this.maplibreMap.setStyle("https://demotiles.maplibre.org/style.json")
```

Comment:
Could you include the examples similar to the other guide?

Because I don't think this compiles.

Reply:
ok

Reply:
Should I provide a working tile both in TestApp and code snippet here?

Reply:
@louwers  provided included links of examples , instead of hardcode

Reply:
rendering correctly in locatserver